app/app.go

```diff
@@ -926,6 +943,30 @@ func (app *App) RegisterTendermintService(clientCtx client.Context) {
 	tmservice.RegisterTendermintService(app.BaseApp.GRPCQueryRouter(), clientCtx, app.interfaceRegistry)
 }
 
+// RegisterStreamingService is used to register a streaming service into the App
+func (app *App) registerStreamingService(appOpts servertypes.AppOptions) {
+	app.Logger().Info(fmt.Sprintf("Streaming Service Config options: "+
+		"\n%s/config/app.toml"+
+		"\n----------------------------------"+
+		"\nset streaming.enable true"+
+		"\nset streaming.service to one of %s\n",
+		os.Getenv("PIO_HOME"), appstreaming.ConfigOptions()))
```

Comment:
This is probably too much, and I find it confusing. It took me a while to realize that it was logging what the config would need to be in order to enable streaming as opposed to what it is. That's best left for documentation though. All that's really desired here is the list of keys (services) that you could set `streaming.service` to. Just something like this:
```suggestion
	app.Logger().Info(fmt.Sprintf("Available streaming services: %s", appstreaming.ConfigOptions()))
```

Other notes about this:
1. The `app.toml` file isn't necessarily where this is configured. It could be any of the config files. We should be directing users to the `custom.toml` for this now anyway.
2. `os.Getenv("PIO_HOME")` only gets the home directory when it's defined by an environment variable. But it can also be set by either a `--home` flag or the default. I'm not sure you can properly get the home directory from the info you have in here.